docs/pages/project/contributing/contributing-docs.md

```diff
@@ -296,6 +296,14 @@ The output of the code snippet would be:
       Thing 3
 ```
 
+### Editable Intra-page Table of Contents Toolbar
+
+Control the display of this intra-page navigator with either page level or layout level frontmatter variables:
+
+`display-toolbar`
+
+Set to `true` (make "editable" toolbar visible) or `false` (hide "editable" toolbar)
+
 ### `if` condititional
```

Comment:
Perhaps not a merge blocker, however I suggest adjusting this word to fix this minor grammar error
```suggestion
### `if` conditional
```

Reply:
Oh, thank you. Very good.